dashbuilder-client/dashbuilder-navigation-client/src/main/resources/org/dashbuilder/client/navigation/resources/i18n/NavigationConstants.properties

```diff
@@ -29,24 +29,38 @@ save=Save
+navCarouselDragComponentNavGroupHelp=The navigation group you select is used to retrieve a list of perspectives, which are displayed as an slideshow carousel - one perspective per slide.
 navTabListDragComponentEmptyError=Please, make sure the configured group contains at least one user created perspective.
 navTilesDragComponentEmptyError=Please, make sure the group configured is not empty.
 navCarouselDragComponentEmptyError=Please, make sure the group configured contains at least one user created perspective.
 navTabListDragComponentDeadlockError=This tab is not displayed to avoid an infinite loop. In general the perspectives in the navigation group you selected for this component must not contain navigation components which refer (directly nor indirectly) to the perspectives in that same group.
 navTilesDragComponentDeadlockError=The content is not displayed to avoid an infinite loop. In general the perspectives in the navigation group you selected for this component must not contain navigation components which refer (directly nor indirectly) to the perspectives in that same group.
 navCarouselDragComponentDeadlockError=This carousel slide is not displayed to avoid an infinite loop. In general the perspectives in the navigation group you selected for this component must not contain navigation components which refer (directly nor indirectly) to the perspectives in that same group.
-navItemSelectorHint=-Select item -
-navItemSelectorHelp=The item that will be used as the root element of the navigation component
-navItemSelectorLabel=Navigation item
-navItemSelectorHeader=Navigation item selector
+targetDivIdPerspectiveDeadlockError=The content is not displayed to avoid an infinite loop. In general a perspective must not contain references (directly nor indirectly) to itself.
 openNavItem=Open {0}
 gotoNavItem=Go to {0}
 showNavItem=Show {0}
-
+navConfigHeader=Navigation component configuration
+navGroupLabel=Navigation group
+navGroupHelp=The group used as the root element of the navigation component.
+navGroupSelectorHint=-Select group -
+defaultItemLabel=Default item
+defaultItemHelp=The item (e.g. a perspective) the navigation component displays by default. If nothing selected then the first item is taken.
+defaultItemSelectorHint=-Select default item -
+defaultItemsNotFound=The selected group has no items
+targetDivLabel=Target div
+targetDivHelp=The div component beloging to the current layout where a runtime perspective item is shown once clicked.
+targetDivSelectorHint=-Select target div -
+targetDivsNotFound=No target divs defined
+navItemEditorPerspectiveHelp=The perspective to display when this navigation item is clicked from any navigation component: menu bar, tile navigator, tab list, tree, ...
+navItemEditorGroupHelp=The navigation group to be used by the target perspective''s navigation components. Notice, this setting is only enabled for user created perspectives.
```

Comment:
1) Remove "Notice, " to make it bit shorter.
2) Also you're mentioning "target perspective", but it might not be clear it's the perspective selected in the dropdown above it.

Perhaps we could prevent one kind of "Does nothing" configuration, when the target perspective has no navigation components? In such case the message could be "... This setting only makes sense when perspective selected above contains navigation components."

Reply:
Done